src/Microsoft.DotNet.GlobalTools.Sdk/Microsoft.DotNet.GlobalTools.Sdk.nuspec

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Provides additional support to .NET Core teams producing global CLI tools. This package is only intended for internal Microsoft use.</description>
+    <copyright>$copyright$</copyright>
+    <packageTypes>
+      <!-- This project needs to be MSBuildSdk because its targets need to influence the way /t:Restore works -->
+      <packageType name="MSBuildSdk" />
```

Comment:
Does Razor Sdk need one of these?

Reply:
Depends. Do you expect users to reference razor as a PackageReference or only as an Sdk?

If you add this, UI tooling in VS for PackageReference will break.

Reply:
Preferably both. We use it as a PackageReference in other repos in aspnet

Reply:
If NuGet UI tooling wasn't a concern, I would say you should add this to your .nuspec

```xml
  <packageTypes>
      <packageType name="Dependency" /> <!-- PackageReference usage -->
      <packageType name="MSBuildSdk" /> <!-- Sdk usage -->
  </packageTypes>
```

But that might be dangerous because the NuGet team is thinking of breaking packages with multiple types.

See https://github.com/NuGet/Home/issues/6298 and https://github.com/NuGet/Home/issues/6484 for context.